nuxt.config.js

```diff
@@ -183,15 +183,14 @@ export default async () => {
         */
         hooks: {
             build: {
-                async before(builder) {
+                async before (builder) {
                     await whitelist(builder);
                 },
-                async done(builder) {
+                async done (builder) {
                     await staticFiles(builder);
                     await images(builder);
                     await fonts(builder);
-                    if (process.env.NODE_ENV && process.env.NODE_ENV === 'production')
-                        await sitemap(join(builder.nuxt.options.buildDir, 'dist', 'static'));
+                    if (process.env.NODE_ENV && process.env.NODE_ENV === 'production') { await sitemap(join(builder.nuxt.options.buildDir, 'dist', 'static')); }
```

Comment:
```suggestion
                    if (process.env.NODE_ENV && process.env.NODE_ENV === 'production') {
                        await sitemap(join(builder.nuxt.options.buildDir, 'dist', 'static'));
                    }
```
This line has become very long

Reply:
Yup, that's true! Applied your suggestion 😋